content/glossary.md

```diff
@@ -34,88 +35,81 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
 
 ---
 
-**Sampling**  
-Is the mechanism that determines how often the system will trace a request.  
-There are  3 types of sampling:  
-
-1. **Always sample:**  
-  
-    - With this sampling, every single request is traced.  
-		
-2. **Probabilistic sampling:**  
-	
-	- Here a probability is set e.g specify 1 in 10000 or 0.0001 and the system will sample only 1 in 10000 requests. This entirely depends on your QPS.  
-		
-3. **Never sample:**  
-	
-	- Here you’ve asked that the system never traces any request.  
+**Sampling**
+
+Sampling determines how often requests will be traced.
+There are three types of sampling made available from OpenCensus libraries:  
+
+* **Always sample:** With this sampling, every single request is traced.  
+* **Probabilisticly sample:** A probability is set (e.g 0.0001) and the libraries will sample only 1 in 10000 requests.
```

Comment:
nit: consider changing to "A probability is set (e.g 0.0001) and the libraries will sample according to that probability (e.g 1 in 10000 requests)."

Reply:
Done.